server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -1223,7 +1223,7 @@ public void testRunConcurrentSnapshots() {
 
     private RepositoryData getRepositoryData(Repository repository) {
         final PlainActionFuture<RepositoryData> res = PlainActionFuture.newFuture();
-        repository.getRepositoryData(res);
+        repository.getRepositoryData(deterministicTaskQueue::scheduleNow, res);
```

Comment:
The PR description says
> does not adjust any non-test callers to use it yet.

But even for tests, this seems to be the only place where we don't use a Direct executor?

Reply:
Correct :)